forestserviceprototype/specialuseform/templates/specialuseform/submitted_permit.html

```diff
@@ -1,8 +1,30 @@
 {% extends "specialuseform/base.html" %}
 
 {% block content %}
-  <h1>Thank you!</h1>
-  <p class="usa-font-lead">
-    We've got it! We'll be in touch.
-  </p>
+  <h1>Received Application</h1>
```

Comment:
I think it should be out of it, because again, this would be the page that they would be coming back to to check their status right?